triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -188,14 +200,12 @@ object Trigger extends StrictLogging {
       case Right(DValueSignature(TApp(ty @ TTyCon(tyCon), tyArg), _, _)) =>
         val triggerIds = TriggerIds(tyCon.packageId)
         if (tyCon == triggerIds.damlTriggerLowLevel("Trigger")) {
-          logger.debug("Running low-level trigger")
```

Comment:
No longer needed as we log the trigger level as part of the root `Runner` logging context